pkg/controllermanager/controller/shoot/maintenance/reconciler.go

```diff
@@ -621,3 +631,23 @@ func ExpirationDateExpired(timestamp *metav1.Time) bool {
 	}
 	return time.Now().UTC().After(timestamp.Time) || time.Now().UTC().Equal(timestamp.Time)
 }
+
+// updateToContainerd updates the CRI of a Shoot's worker pools to containerd
+func updateToContainerd(log logr.Logger, shoot *gardencorev1beta1.Shoot, reason string) ([]string, error) {
+	var reasonsForUpdate []string
+
+	for i, worker := range shoot.Spec.Provider.Workers {
+		workerLog := log.WithValues("worker", worker.Name)
+
+		if worker.CRI == nil || worker.CRI.Name != gardencorev1beta1.CRINameDocker {
+			continue
+		}
+
+		shoot.Spec.Provider.Workers[i].CRI.Name = gardencorev1beta1.CRINameContainerD
+
+		workerLog.Info("CRI will be updated to containerd", "reason", reason)
```

Comment:
Do we need this log? We don't log anything in the other case (1.26->1.27 upgrade), and it's also quite clear what happens when the maintenance force-upgrade takes place (which can be observed via `Shoot` events).

Reply:
ok, I removed it 👍